draft-ietf-quic-transport.md

```diff
@@ -2527,6 +2546,26 @@ Reordering might cause frames to be received after closing, see
 {{state-hc-remote}}.
 
 
+## Solicited State Transitions
+
+If an endpoint is no longer interested in the data being received, it MAY send a
+STOP_SENDING frame on a stream in the "open" or "half-closed (local)" state to
+request closure of the stream in the opposite direction.  This typically
+indicates that the receiving application is no longer reading from the stream,
+but is not a guarantee that incoming data will be ignored.
+
+STREAM frames received after sending STOP_SENDING are still counted toward the
+connection and stream flow-control windows.  Even though these frames will be
+discarded, because they are sent before their sender receives the STOP_SENDING,
+the sender will consider the frames to count against its flow-control windows.
+
+Upon receipt of a STOP_SENDING frame on a stream in the "open" state, an
```

Comment:
is there a reason this requirement is only in open state and not in half-closed remote as well? 

This whole paragraph seems strangely worded, it just basically says, "Hey you can send a RST stream in response to a STOP_WAITING" any time. Does it add a lot of value to make QUIC_RECEIVED_RST a special case?